.travis.yml

```diff
@@ -2,10 +2,12 @@ sudo: false
 
 language: go
 
+env:
+  global:
+  - GO15VENDOREXPERIMENT=1
```

Comment:
https://github.com/kubernetes/contrib/pull/374 from @mikedanese moved us to this mechanism. I agree, it's not the best...

Then again, there are other efforts to break this repo apart further which makes this distinction less meaningful...